db/schema.ts

```diff
@@ -176,6 +176,7 @@ export const JobsTable = pgTable("jobs", {
   contactId: uuid("contact_id").references(() => ContactsTable.id),
   status: JobStatusesEnum("status").default("OPEN"),
   reason: text("reason"),
+  externalUrl: varchar("external_url"),
```

Comment:
Add this column to our python model too.

Then, for all instances where we create a job, make sure we have this column filled